### ISSUE TITLE
No backlight on keyboard

### ISSUE THREAD

**Issue by Kabouik** [sailfish-on-fxtecpro1/droid-config-t5#22]
Pretty much what the title says, unless I missed an option somewhere.

**Comment by Kabouik** [comment]
Interestingly, the Caps lock LED works.

**Comment by Kabouik** [comment]
[Wolda on TMO](https://talk.maemo.org/showpost.php?p=1564033&postcount=60) found that keyboard backlight can be enabled manually using `echo 1 > /sys/class/leds/keyboard-backlight/brightness`.

Based on this, here is a temporary workaround using [qCommand](https://openrepos.net/content/coanteen/qcommand) from Openrepos:

Add two new commands in qCommand and enable "cover action" for each

1. "Keyboard backlight on":
`echo 1 > /sys/class/leds/keyboard-backlight/brightness`

2. "Keyboard backlight off":
`echo 0 > /sys/class/leds/keyboard-backlight/brightness`

Now one can spam the qCommand cover action and it turns the backlight on and off. It is not even necessary to select each cover action individually because they replace each other when one is used.

Example:

![](https://reho.st/0x0.st/zdHl.png)

Using the above action turns keyboard backlight on and changes the default cover action to the following, which will do the opposite: 

![](https://reho.st/0x0.st/zdHU.png)

**Comment by Kabouik** [comment]
Thanks to @elros34, we can have proper (and automatic) keyboard backlight!

Until this is included in the Pro¹ SFOS port, here is how to get it working:

```
# Fetch the mce keyboard backlight config file from elros34's repository
cd /etc/mce
devel-su curl -L -o 91-keyboard-backlight.ini https://raw.githubusercontent.com/elros34/droid-config-moto_msm8960_jbbl/master/sparse/etc/mce/91-keyboard-backlight.ini
```

A Lipstick restart may be required.

**Comment by Michal-Szczepaniak** [comment]
well PR would be nice but adding it would be simple as well

**Comment by Kabouik** [comment]
I didn't do a PR because I can't really have a local git copy here at work, sorry.

**Comment by Kabouik** [comment]
Forgot to close this. Fixed by f3a368fe8f09d7eaa2ee55ef3cafd745d3c53104.
